lib/alerts/overdue_training_alert_manager.rb

```diff
@@ -29,7 +29,7 @@ def manage_alert(course, student)
     course.training_modules.each do |training_module|
       due_date_manager = TrainingModuleDueDateManager.new(course: course, user: student,
                                                           training_module: training_module)
-      overdue = true if due_date_manager.overdue?
+      overdue = true if due_date_manager.overdue? && training_module.is_training?
```

Comment:
Maybe instead, it should do `next if training_module.is_training?` at the top of the `each` block. That way, no status will get inserted into the `status` hash for the other module types. `status` here ends up getting converted to a table in the email, so we probably don't want to include the non-trainings at all: https://dashboard-testing.wikiedu.org/rails/mailers/overdue_training_alert_mailer/message_to_student